contrib/jemalloc-cmake/include_linux_aarch64/jemalloc/internal/jemalloc_internal_defs.h.in

```diff
@@ -35,7 +35,7 @@
  */
 #define CPU_SPINWAIT
 /* 1 if CPU_SPINWAIT is defined, 0 otherwise. */
-#define HAVE_CPU_SPINWAIT 9
+#define HAVE_CPU_SPINWAIT 0
```

Comment:
I wonder what is the reason for this change...

Reply:
You should wonder before, in https://github.com/ClickHouse/ClickHouse/commit/82f2b1a986965fc850188460f4873566101692a0#diff-ee00bf922b046bbd74fef9b9dd42e366L44-R38 :)
This is a fix for a typo that I made in #11163, this is reflected in commit message for 6704c42a9ec79828057c4b2bcb813512d5e98f1e